.github/workflows/integration-tests-on-release.yaml

```diff
@@ -194,7 +194,7 @@ jobs:
     - name: TestCephUpgradeSuite
       run: |
        export DEVICE_FILTER=$(lsblk|awk '/14G/ {print $1}'| head -1)
-       go test -v -timeout 1800s -run CephUpgradeSuite github.com/rook/rook/tests/integration
+       go test -v -timeout 1800s -run CephUpgradeSuite/TestUpgradeRookToMaster github.com/rook/rook/tests/integration
```

Comment:
(asking for my understanding) Why do we want to run only RookToMaster upgrade tests and not the other ceph upgrade tests in this suite?

Reply:
Because the other tests are run on a daily basis (nightly jobs) and this one runs on PRs and release.